torchvision/models/quantization/inception.py

```diff
@@ -0,0 +1,254 @@
+import sys
+import warnings
+from collections import namedtuple
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision.models.inception
+from torch import Tensor
+from torch.jit.annotations import Optional
+from torchvision.models.utils import load_state_dict_from_url
+
+
+inception_module = sys.modules["torchvision.models.inception"]
```

Comment:
Can you import this module as a regular import? Why is this being imported in this manner?

Reply:
This is because if I do regular import such as:
import torchvision.models.inception as inception_module

it would give error:
AttributeError: module 'torchvision' has no attribute 'models'

Reply:
Using from works: 
from torchvision.models import inception as inception_module

Reply:
Using from works for inception but doesn't work for googlenet since googlenet.py has the function with the same name, which will cause error:
```
AttributeError: 'function' object has no attribute 'BasicConv2d'
```